source/install/cloud-changelog.md

```diff
@@ -4,12 +4,74 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
+- [Release 2022-06-15](#release-2022-06-15)
 - [Release 2022-05-26](#release-2022-05-26)
 - [Release 2022-05-12](#release-2022-05-12)
 - [Release 2022-04-28](#release-2022-04-28)
 - [Release 2022-04-13](#release-2022-04-13)
 - [Release 2022-03-30](#release-2022-03-30)
-- [Release 2022-03-16](#release-2022-03-16)
+
+## Release 2022-06-15
+
+### Highlights
+
+#### Workspace Limits
+ - Mattermost is releasing a free-forever Cloud offering.
+   - Added a new API endpoint ``GET /api/v4/usage/posts`` that returns the current usage limits for messages.
+   - Team Limit changes now propagate automatically.
+   - Added a new endpoint ``DELETE /api/v4/teams/{team_id:[A-Za-z0-9]+}/except`` which archives all teams except for the ``team_id`` passed to the endpoint.
+   - A server configured to use the Cloud Freemium limits now guards against enabling more integrations than the configured limit.
+Users are restricted from creating teams when at or above the teams limit for their plan.
+   - Fixed API ``GET /api/v4/usage/posts`` to include user posts only.
+   - Improved performance of API ``GET /api/v4/usage/posts`` by enabling caching of posts count.
+   - Users will be prevented from unarchiving teams if they're at or over the teams limit for their plan.
+
+#### Calls (Beta)
+ - Native voice calling and screen sharing is now available. This is a Channels-specific integration.
+
+#### Apps Bar (Beta)
+ - Channel header is now decluttered when a new configuration setting via **System Console > Experimental > Features > Enable App Bar** is enabled, the channel header will be decluttered to make it more obvious how to access Calls, Playbooks, and Boards when viewing a channel. All channel header icons registered by plugins will be moved to the new Apps Bar when the configuration setting is enabled. We recommend enabling the Apps Bar for servers with Calls enabled since the Apps Bar helps make space for the dedicated **Start Call** button in the channel header.
+
+#### Collapsed Reply Threads (General Availability)
+ - [Collapsed Reply Threads](XXXXXX link to CRT GA blog post) is now generally available. Please reference [this article](link to KB article XXXXXX) for more information and guidance for enabling the feature.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a modal to allow users to switch subscription plans pre-trial.
+ - Applied new designs for the Login screen.
+ - Changed some tooltips to appear when focused instead of just on hover.
+ - The legacy ``Enable post formatting`` setting is now hidden when the Message Formatting Toolbar is enabled.
+ - Updated Apps Framework to version 1.1.0 to add improved logging.
+
+#### Administration
+ - The default for ``ServiceSettings.CollapsedThreads`` has been changed to "always_on". This does not affect existing configurations where this value is already set to some other value.
+ - To enable or disable the App Bar, a new config setting ``ExperimentalSettings.EnableAppBar`` was added. 
```

Comment:
Does not apply to Cloud (at least couldn't find such an option in the System Console)